config/config.go

```diff
@@ -116,7 +116,7 @@ var (
 			Active:                true,
 			HeartbeatInterval:     10 * time.Second,
 			HTTPStatsPort:         8080,
-			HTTPAdminPort:         9009,
+			HTTPAdminPort:         0,
```

Comment:
would this cause some production service (like API nodes) on port 9009 not available? like we are using https://xx.xx.xx.xx:9009 for something

Reply:
Yes. It should be disabled by default